content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### 📈 Advanced admin capabilities
+<br />
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+We know that upgrading Sourcegraph can be a time-intensive process, especially if you fall a few versions behind. Sourcegraph 4.0 supports multi-version upgrades so you can upgrade to 4.0 directly from version 3.20 or higher.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20upgrades%20final',
+    mp4: 'blog/release-post/4.0/Multi-version%20upgrades%20final'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Upgrade to 4.0 directly from version 3.20 or higher with multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/analytics"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Understand usage and quantify the value of using Sourcegraph with in-product analytics
+Our new in-product analytics help admins to understand user engagement, measure efficiency in terms of time saved, and calculate the value of Sourcegraph and its features. 
+
+These analytics work across all of our products and provide ROI-based metrics such as:
+
+- The number of searches developers have run in Sourcegraph, and how many hours of development time have been saved as a result.
+- How many code insights have been created, viewed, and interacted with.
+- The number of changesets that have been created and merged via batch changes, and how much time has been saved as a result.
+- How frequently code navigation has been used by developers to more quickly understand code.
+- How often IDE and Browser extensions are used across your organization, and how much time has been saved as a result.
+- The number of navigation events and the percentage of your repositories that allow precise code navigation
+
+Admins can select a specific date range and customize the assumpted to align with how the platform is used. 
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search.png"
+    alt="In-product analytics"
+    caption="Docs"
+    link="https://docs.sourcegraph.com/admin/analytics"
+    linkIcon={true}
+  />
+</div>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/observability/opentelemetry"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Easily export traces using OpenTelemetry
```

Comment:
@quinnhare you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.

Reply:
Looks good to me.